public/docs/_examples/displaying-data/dart/lib/app_component_2.dart

```diff
@@ -0,0 +1,30 @@
+// #docregion
+library displaying_data.app_component_2;
+
+import 'package:angular2/angular2.dart';
+
+const List<String> _heroes = const [
+  'Windstorm',
+  'Bombasto',
+  'Magneta',
+  'Tornado'
+];
+
+@Component(
+    selector: 'my-app',
+    template: '''
+<h1>{{title}}</h1>
+<h2>My favorite hero is: {{myHero}}</h2>
+<p>Heroes:</p>
+<ul>
+  <li *ng-for="#hero of heroes">
+    {{ hero }}
+  </li>
+</ul>''',
+    directives: const [NgFor])
```

Comment:
That's good news. It seemed really odd to have to in the first place. I thought maybe it was for better performance somehow.